### ISSUE TITLE
Channel export onto external drive failing

### ISSUE THREAD

**Issue by lauradanforth** [learningequality/kolibri#10905]
## Observed behavior
The channel was exporting for a while and then eventually showed that it had failed
…

## Errors and logs
[kolibri.txt](https://github.com/learningequality/kolibri/files/11858264/kolibri.txt)

Please wrap errors in triple backticks for clean formatting like this:
```
Kolibri Modules: kolibri.plugins.device.app registered
actions.js:210 [INFO: kolibri/core/assets/src/state/modules/core/actions.js] Notifications set.
:8080/en/device/#/content:1 Autofocus processing was blocked because a document already has a focused element.
:8080/device/api/device_channel/4489f959ec9a40c0b6c571bdd2f398df/?include_fields=total_resources%2Ctotal_file_size%2Con_device_resources%2Con_device_file_size%2Cnew_resource_count%2Cnew_resource_total_size:1     Failed to load resource: the server responded with a status of 404 (Not Found)
api-resource.js:1034 Request error: Not Found, 404 for GET to /device/api/device_channel/4489f959ec9a40c0b6c571bdd2f398df/ - open for more info
handlers.js:171 ^^^ 404 (Not Found) error returned while requesting "/device/api/device_channel/4489f959ec9a40c0b6c571bdd2f398df/..." is an expected response.
:8080/device/api/device_channel/624e09bb5eeb4d20aa8de62e7b4778a0/?include_fields=total_resources%2Ctotal_file_size%2Con_device_resources%2Con_device_file_size%2Cnew_resource_count%2Cnew_resource_total_size:1     Failed to load resource: the server responded with a status of 404 (Not Found)
api-resource.js:1034 Request error: Not Found, 404 for GET to /device/api/device_channel/624e09bb5eeb4d20aa8de62e7b4778a0/ - open for more info
handlers.js:171 ^^^ 404 (Not Found) error returned while requesting "/device/api/device_channel/624e09bb5eeb4d20aa8de62e7b4778a0/..." is an expected response.
:8080/device/api/device_channel/ec29f4cc20a8437d844a60297c2ffd07/?include_fields=total_resources%2Ctotal_file_size%2Con_device_resources%2Con_device_file_size%2Cnew_resource_count%2Cnew_resource_total_size:1     Failed to load resource: the server responded with a status of 404 (Not Found)
api-resource.js:1034 Request error: Not Found, 404 for GET to /device/api/device_channel/ec29f4cc20a8437d844a60297c2ffd07/ - open for more info
handlers.js:171 ^^^ 404 (Not Found) error returned while requesting "/device/api/device_channel/ec29f4cc20a8437d844a60297c2ffd07/..." is an expected response.
DevTools failed to load source map: Could not load content for chrome-extension://bmnlcjabgnpnenekpadlanbbkooimhnj/browser-polyfill.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
DevTools failed to load source map: Could not load content for chrome-extension://lggdbpblkekjjbobadliahffoaobaknh/content.72a61e7a.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
2DevTools failed to load source map: Could not load content for chrome-extension://ppbfhhgcdehhbjigifndnfmcihegokbb/globals/custom-elements.min.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
```
-->

…

<img width="1168" alt="image" src="https://github.com/learningequality/kolibri/assets/22602993/59c598fa-b7e1-4175-98d2-88ba59e09333">

…

## Context

Tell us about your environment, including:
 * Kolibri version 0.16.0a15
 * Operating system Mac OS
 * Browser Chrome

…

**Comment by lauradanforth** [comment]
Relatedly, while it was in progress, it was showing that it was over 100% complete 

![image](https://github.com/learningequality/kolibri/assets/22602993/5d4751b5-3bf9-4ee1-ab81-fc99cdce6035)

**Comment by rtibbles** [comment]
I think the main issue we can handle here has probably already been resolved in #10825.

Looking at the logs, there is a specific issue with a release of MacOS Catalina that can cause errors when attempting to make parallel writes to a USB mounted drive. There's not much we can do about that, and is a genuine error.

The progress updating is logged here #5501.

The rest of this should be cleaned up by #10894
